Examples/Cocoa/Sources/Objective_C/User.m

```diff
@@ -440,7 +440,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder
     _bio = [aDecoder decodeObjectOfClass:[NSString class] forKey:@"bio"];
     _counts = [aDecoder decodeObjectOfClasses:[NSSet setWithArray:@[[NSDictionary class], [NSNumber class]]] forKey:@"counts"];
     _createdAt = [aDecoder decodeObjectOfClass:[NSDate class] forKey:@"created_at"];
-    _emailFrequency = [aDecoder decodeIntegerForKey:@"email_frequency"];
+    _emailFrequency = (UserEmailFrequency)[aDecoder decodeIntegerForKey:@"email_frequency"];
```

Comment:
These casts are necessary for when warnings are enabled in the Pinterest builds.